src/mono/wasi/build/WasiApp.targets

```diff
@@ -101,7 +101,7 @@
     </PropertyGroup>
 
     <PropertyGroup>
-      <_ToolchainMissingErrorMessage Condition="'$(WASI_SDK_PATH)' == '' or !Exists('$(WASI_SDK_PATH)/VERSION24')">Could not find wasi-sdk. Install wasi-sdk and set %24(WASI_SDK_PATH) . It can be obtained from https://github.com/WebAssembly/wasi-sdk/releases</_ToolchainMissingErrorMessage>
+      <_ToolchainMissingErrorMessage Condition="'$(WASI_SDK_PATH)' == '' or !Exists('$(WASI_SDK_PATH)/VERSION24')">Could not find wasi-sdk version 24 or newer. Install wasi-sdk and set %24(WASI_SDK_PATH) . It can be obtained from https://github.com/WebAssembly/wasi-sdk/releases</_ToolchainMissingErrorMessage>
```

Comment:
Do we need to tell that `VERSION24` needs to be created too ? Or We invent better version check ?
Note that `Exists('$(WASI_SDK_PATH)/VERSION24')` one-liner is in many places right now.

Reply:
> Do we need to tell that `VERSION24` needs to be created too ?

Randomly came across this while attempting to figure out why .NET 9 RC 2 keeps claiming my WASI SDK cannot be found. One `touch ~/wasi-sdk/VERSION24` later and everything works. Not sure whether I missed this elsewhere, but I certainly didn't find it elsewhere.